src/views/v2/day/event/featured-image.php

```diff
@@ -28,14 +28,18 @@
 		rel="bookmark"
 		class="tribe-events-calendar-day__event-featured-image-link"
 	>
-		<div class="tribe-events-calendar-day__event-featured-image tribe-common-c-image tribe-common-c-image--bg">
-			<div
-				class="tribe-common-c-image__bg"
-				style="background-image: url('<?php echo esc_url( $event->thumbnail->full->url ); ?>');"
-				role="img"
-				aria-label="<?php echo esc_attr( get_the_title( $event->ID ) ); ?>"
-			>
-			</div>
-		</div>
+		<img
+			src="<?php echo esc_attr( $event->thumbnail->full->url ); ?>"
```

Comment:
:no_entry_sign: Wrong escaping function. href and src attributes should be escaped by `esc_url()`, not by `esc_attr()`.

<sub>hash: 9f7494cf51bd11c0a3cc8c278230b605</sub>